gravitee-plugin-core/src/main/java/io/gravitee/plugin/core/enums/PluginFilesType.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.plugin.core.enums;
+
+/**
+ * @author Yann TAVERNIER (yann.tavernier at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+public enum PluginFilesType {
+    SCHEMAS("schemas"),
+    DOCS("docs"),
+    SUBSCRIPTION_SCHEMAS("schemas/subscriptions");
+
+    private final String directory;
+
+    PluginFilesType(String directory) {
+        this.directory = directory;
+    }
+
+    public String getDirectory() {
+        return directory;
+    }
```

Comment:
maybe an enum is too rigid and enforce coupling with implementation (here we do not know subscriptions can exist).

Maybe we should keep strings as parameter ? 

to discuss